### DISCUSSION TITLE
anybody available for mentorship?

### DISCUSSION THREAD

**Discussion by ranran2121** [Leaflet/Leaflet#9439] [General]
Hi all,

my name is Francesca and I am a self-taught developer aiming to become a GIS developer... so I have just started exploring this beautiful package :). 
Would any of the more experienced contributors like to become my mentor for a while, in order to accelerate my learning process? Thank you in advance :)

**Comment by jonkoops** [comment]
Sorry, but we're already short at hands here and I don't think any of the regular maintainers will be able to help you. Best of luck!

**Reply by ranran2121** [reply]
ops, I am afraid I didn't make it clear that I would be honored to hep you in maintaining :)

**Reply by jonkoops** [reply]
Feel free to take any issues marked as 'help wanted' that is the best way to help us out.
